### DISCUSSION TITLE
Linux package manager

### DISCUSSION THREAD

**Discussion by c60cb859** [williamboman/nvim-lsp-installer#540] [Q&A] [answered]
Hi

Would it make sense to make an installer for using the build package manager in the linux system?
Like for me it would use pacman for installing language servers, removing the requirement for having any dependencies as long as the package mirror has the server.

**Comment by williamboman** [comment, marked as answer]
Hello! This is not on the roadmap. I'd personally not want to add it to core, but it could potentially be an interesting use case for the future, when internals are more extendable.

As for avoiding dependencies, are there any specific examples you have in mind? The requirements for the base installer tools is pretty minimum (basically just curl or wget, gzip, tar, unzip). As for avoiding runtime requirements (e.g. Node), I'm not sure a system package manager would help (unless it vendors the runtime)?

**Reply by c60cb859** [reply]
You are right, I was annoyed that Pyright required npm and node, but you are right that this is runtime dependencies, so using pacman for installing pyright would give me those packages as well.
